### PR TITLE
Ember 1.13 upgrade

### PR DESCRIPTION
**Pull request by bl4ckm0r3**
I have forked your repo and upgraded `ember` to `v1.13` and fixed deprecations.
Notice that, since repo was not using it, i also removed the dependency with `ember/data`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-lazy-video",
   "description": "Lazy loading videos for Ember apps",
-  "version": "0.2.5",
+  "version": "0.3.2",
```
Comment (poteto) [inline package.json]: Could you please revert the change to `version`? I'd like to control the version numbers when I update the next release

**Comment by poteto** [comment]
Thanks for this!

File: package.json (hunk 1 above)
Comment (bl4ckm0r3) [inline package.json]: i think i completely forgot about this comment :P sorry i squashed commits and reverted the `version` to `0.2.5`

Comment (bl4ckm0r3) [inline package.json]: and thank you! for this!

**Comment by Dhaulagiri** [comment]
I'll open a more updated version of this PR shortly
